components/server/ee/src/workspace/workspace-factory.ts

```diff
@@ -47,6 +46,8 @@ export class WorkspaceFactoryEE extends WorkspaceFactory {
                 throw new Error("Can only prebuild workspaces with a commit context")
             }
 
+            const { project, branch } = context;
```

Comment:
Do you expect to use `project` or `branch` somewhere else in the code below?

Otherwise, you could avoid the single-use local variables by using `context.project` and `context.branch` directly in the `storePrebuiltWorkspace()` call below.